book/src/crosslink/changes-to-zebra.md

```diff
@@ -0,0 +1,16 @@
+# Changes to Zebra
+
+One goal while extending upstream [`zebra`](https://github.com/ZcashFoundation/zebra) was to minimize changes to existing code and introduce as much logic as possible into separate crates and `tower` services
+
+## Crosslink Crates & Services
+
+- `zebra-crosslink`: This crate provides the "TFL service" via `TFLServiceHandle` [^1].
+- `zebra-crosslink-chain`: Contains Crosslink-specific protocol parameters and chain state types.
+
+The TFL service is launched via the `zebrad start` subcommand along with other zebra internal services. This service is the connection between all of Zebra's pre-existing functionality and the BFT protocol, provided by the [Malachite BFT Engine](https://github.com/informalsystems/malachite).
```

Comment:
It may be worth mentioning that `zebrad start` is the default mode of operation if you just run a plain `zebrad`